### ISSUE TITLE
[SF][a11y][ui5-select] Currently focused option in the drop downs of Select listboxes not recognized by JAWS

### ISSUE THREAD

**Issue by zhangliangyi** [SAP/ui5-webcomponents#5494]
### **Bug Description**
The current focused option in drop down opened by `ui5-select` is not recognized by screen reader.

### **Expected Behavior**
Screen reader should announce the current focused option.

### **Steps to Reproduce**
1. Go to https://sap.github.io/ui5-webcomponents/playground/components/Select/
2. Press Enter/Space to expand the drop down in a select component
3. Check if the text of current focused option is announced by screen reader or not

### **Context**
- UI5 Web Components version: 1.5.0
- OS/Platform: Mac OS
- Browser: Edge
- Affected component: ui5-select

### **Log Output / Stack Trace / Screenshots**
```
{...}
```
### **Priority**
- [ ] Low
- [ ] Medium
- [x] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: SF
- Business impact: {...}

**Comment by gmkv** [comment]
Hi @SAP/ui5-webcomponents-topic-p 
The Select component's dropdown options do not get announced by JAWS.

To reproduce:
1. Load https://sap.github.io/ui5-webcomponents/playground/main/pages/Select/
2. Focus a Select and open it's dropdown with Space/Enter.
3. The currently focused option is not announced. Focusing the other options with the keyboard will also not be announced.

Could you have a look?

**Comment by gmkv** [comment]
Related to #3662

**Comment by don-obrien** [comment]
@PetyaMarkovaBogdanova .  can you give an update on the state of this, and when we can expect it.

**Comment by PetyaMarkovaBogdanova** [comment]
Hi @don-obrien ,
the fix is in code-review, I am still polishing some details and expect it to be merged till the end of the week.

Best,
Petya Markova.

**Comment by GongRichard** [comment]
Hi @ilhan007 , 
This issue still exists, would you please reopen this issue?
When opening the listbox, the "currently focused" option is not read out
However as we arrow down and up, the options are read thereafter. 
Thanks,
Richard

**Comment by GongRichard** [comment]
Hi @PetyaMarkovaBogdanova and @ilhan007 ,
UI5 Web Components version: 1.11.0
Thanks,
Richard

**Comment by ksblklu** [comment]
Hi @PetyaMarkovaBogdanova 
Verify this issue with JAWS 2022, NVDA 2022, the selected option is still not read when opening the ui5-select dropdown:
<img width="1691" alt="image" src="https://github.com/SAP/ui5-webcomponents/assets/27538066/bc95d98a-c9be-45f4-a69e-1186d94b32b9">

Would you please kindly reopen this ticket and take a look? Thanks!
